nf_core/pipeline-template/conf/modules.config

```diff
@@ -15,14 +15,14 @@ process {
     publishDir = [
         path: { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
         mode: params.publish_dir_mode,
-        saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        pattern: "{!versions.yml}"
```

Comment:
I thought this was a shell pattern. Since when has this been possible?

Reply:
It's not working for me:
```nextflow
workflow {
  TASK()
}

process TASK {
    input:

    script:
    """
    touch versions.{yml,txt}
    """

    output:
    path "versions.yml"
    path "versions.txt"
    
}
```
```
process.publishDir = [ path: "results", pattern: "{!versions.yml}" ]
```
Nothing is output in this case, except the folder is made.

Reply:
ok let me fix that then

Reply:
yeah, you're right, not working, I should have check more my outputs